DeAceroDLabs/marketplace-frontend
# [DMP-509] Use local storage for shopping cart

## What is done

[[DMP-509] Publicar nombre de usuario en Auth0](https://deacero.atlassian.net/browse/DMP-509)

## How is done

1. Created a custom hook that will be used to store data on local storage.
2. Added the new hook to the ShoppingCart and Products component.
3. The shopping cart view  checks on local storage for items in the shopping cart every time is loaded so on refresh the items won't be lost.

## How to test

1. Run the app `npm start`.
4. Go to `/carrito/productos`.
5. You will see that two products are pre-added to the cart (this is just for testing purposes).
6. Try removing any item from the list.
7. Refresh the page. You will see the list as it was before the refresh.
8. Remove all items and repeat the previous step.

> For resetting the local storage, go to developer tools > Application tab > Local Storage > select the local storage and delete it
